<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagEditorClass</class>
 <widget class="QDialog" name="TagEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TagEditor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/application.png</normaloff>:/icons/application.png</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="label">
     <property name="text">
      <string>File</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/ban.png</normaloff>:/icons/ban.png</iconset>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QPushButton" name="pushButton_Save">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_Correct">
     <property name="text">
      <string>Correct 8bit locale</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_Correct2">
     <property name="text">
      <string>Correct broken UTF</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="lineTitle"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLineEdit" name="lineArtist"/>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QLineEdit" name="lineAlbum"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Artist</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Album</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="3">
    <widget class="QLineEdit" name="lineComment"/>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_Genre">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Genre</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QSpinBox" name="spinBox_Track">
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="spinBox_Year">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Track</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QSpinBox" name="spinBox_Rating">
     <property name="minimum">
      <number>-99</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Rating</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineTitle</tabstop>
  <tabstop>lineArtist</tabstop>
  <tabstop>lineAlbum</tabstop>
  <tabstop>comboBox_Genre</tabstop>
  <tabstop>spinBox_Track</tabstop>
  <tabstop>spinBox_Year</tabstop>
  <tabstop>spinBox_Rating</tabstop>
  <tabstop>lineComment</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
  <tabstop>pushButton_Save</tabstop>
  <tabstop>pushButton_Correct</tabstop>
  <tabstop>pushButton_Correct2</tabstop>
  <tabstop>label</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Save</sender>
   <signal>clicked()</signal>
   <receiver>TagEditorClass</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>TagEditorClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Correct</sender>
   <signal>clicked()</signal>
   <receiver>TagEditorClass</receiver>
   <slot>correct1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Correct2</sender>
   <signal>clicked()</signal>
   <receiver>TagEditorClass</receiver>
   <slot>correct2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
